RealmSwift/MutableSet.swift

```diff
@@ -0,0 +1,548 @@
+     */
+    public func observe(on queue: DispatchQueue? = nil,
+                        _ block: @escaping (RealmCollectionChange<MutableSet>) -> Void) -> NotificationToken {
+        return _rlmSet.addNotificationBlock(wrapObserveBlock(block), queue: queue)
+    }
+
+    // MARK: Frozen Objects
+
+    public var isFrozen: Bool {
+        return _rlmSet.isFrozen
+    }
+
+    public func freeze() -> MutableSet {
+        return MutableSet(objc: _rlmSet.freeze())
+    }
+
+    public func thaw() -> MutableSet? {
+        return MutableSet(objc: _rlmSet.thaw())
+    }
+
+    // swiftlint:disable:next identifier_name
+    @objc class func _unmanagedSet() -> RLMSet<AnyObject> {
+        return Element._rlmSet()
+    }
+}
+
+extension MutableSet where Element: MinMaxType {
+    /**
+     Returns the minimum (lowest) value in the set, or `nil` if the set is empty.
+     */
+    public func min() -> Element? {
+        return _rlmSet.min(ofProperty: "self").map(dynamicBridgeCast)
+    }
+
+    /**
+     Returns the maximum (highest) value in the set, or `nil` if the set is empty.
+     */
+    public func max() -> Element? {
+        return _rlmSet.max(ofProperty: "self").map(dynamicBridgeCast)
+    }
+}
+
+extension MutableSet where Element: AddableType {
+    /**
+     Returns the sum of the values in the set.
+     */
+    public func sum() -> Element {
+        return sum(ofProperty: "self")
+    }
+
+    /**
+     Returns the average of the values in the set, or `nil` if the set is empty.
+     */
+    public func average<T: AddableType>() -> T? {
+        return average(ofProperty: "self")
+    }
+}
+
+extension MutableSet: RealmCollection {
+    /// The type of the objects stored within the set.
+    public typealias ElementType = Element
+
+    // MARK: Sequence Support
+
+    /// Returns a `RLMIterator` that yields successive elements in the `MutableSet`.
+    public func makeIterator() -> RLMIterator<Element> {
+        return RLMIterator(collection: _rlmSet)
+    }
+
+    /// :nodoc:
+    // swiftlint:disable:next identifier_name
+    public func _asNSFastEnumerator() -> Any {
+        return _rlmSet
+    }
+
+    /// The position of the first element in a non-empty collection.
+    /// Identical to endIndex in an empty collection.
+    public var startIndex: Int { return 0 }
+
+    /// The collection's "past the end" position.
+    /// endIndex is not a valid argument to subscript, and is always reachable from startIndex by
+    /// zero or more applications of successor().
+    public var endIndex: Int { return count }
+
+    public func index(after i: Int) -> Int { return i + 1 }
+    public func index(before i: Int) -> Int { return i - 1 }
+
+    /// :nodoc:
+    // swiftlint:disable:next identifier_name
+    public func _observe(_ queue: DispatchQueue?,
+                         _ block: @escaping (RealmCollectionChange<AnyRealmCollection<Element>>) -> Void)
+        -> NotificationToken {
```

Comment:
no newline

Reply:
`List` does the same here, I presume its to preserve some style relating the the amount of max characters per line.